src/endpoint/subscriptions.ts

```diff
@@ -329,7 +329,7 @@ export class SubscriptionsEndpoint extends Endpoint {
 						componentId: item.deviceConfig.componentId,
 						capability: capability,
 						attribute: attributeName,
-						stateChangeOnly: options.stateChangeOnly ? options.stateChangeOnly : true,
+						stateChangeOnly: options.stateChangeOnly !== undefined ? options.stateChangeOnly : true,
```

Comment:
This works but another way you could do this that would also check for `null` (but not `false` or `0`) would be to use the new `??` operator:

```
stateChangeOnly: options.stateChangeOnly ?? : true,
```